src/editors/array.js

```diff
@@ -493,6 +493,13 @@ JSONEditor.defaults.editors.array = JSONEditor.AbstractEditor.extend({
       self.rows[i].delete_button.addEventListener('click',function(e) {
         e.preventDefault();
         e.stopPropagation();
+
+        if (self.jsoneditor.options.prompt_before_delete === true) {
+          if (confirm("Confirm to remove.") === false) {
```

Comment:
Confirmation message could be a little bit more contextually evident. for example:
* Clicking delete button: 'Are you sure you want to delete this node?'
* Clicking delete last row button: 'Are you sure you want to delete the last row in this node?'
* Clicking remove all rows button: 'Are you sure you want to delete all rows in this node?'

Reply:
Optional :)